server.ts

```diff
@@ -46,7 +46,9 @@ const CONFIG = {
   'activity_next_poll_request_ms': parseInt(process.env['CCD_ACTIVITY_NEXT_POLL_REQUEST_MS'], 10) || 5000,
   'activity_retry': parseInt(process.env['CCD_ACTIVITY_RETRY'], 10) || 5,
   'activity_batch_collection_delay_ms': parseInt(process.env['CCD_ACTIVITY_BATCH_COLLECTION_DELAY_MS'], 10) || 1,
-  'activity_max_request_per_batch': parseInt(process.env['CCD_ACTIVITY_MAX_REQUEST_PER_BATCH'], 10) || 25
+  'activity_max_request_per_batch': parseInt(process.env['CCD_ACTIVITY_MAX_REQUEST_PER_BATCH'], 10) || 25,
+  'shutter_redirect_url': process.env['SHUTTER_REDIRECT_URL'] || '',
+  'shutter_redirect_delay': parseInt(process.env['SHUTTER_REDIRECT_DELAY'], 10) || 10
```

Comment:
Could we say 'wait' instead of 'delay'? Just a very little wording suggestion.

Reply:
Addressed 👍